File: test09/sort_words.py
#!/usr/bin/python

import sys
import re

for line in sys.stdin:
    list=line.split(' ')
    list.sort()
    l=''
    
    for i in list:
        if i != '\s' and i != '\n':
            i=i.strip('\n')
            l=l+i+' '
    print(l)

File: test07/sort_words.pl
#!/usr/bin/perl


while ($str=<STDIN>){
@shuzu=split " ",$str;
@shu=();
foreach $shu(@shuzu){
if ($shu){
push(@shu,$shu);
}
}
@shu= sort {$a cmp $b} @shu;
$str= join(' ',@shu);
print "$str\n";
}
